Firestore/core/src/core/sync_engine.cc

```diff
@@ -567,6 +573,74 @@ void SyncEngine::RemoveLimboTarget(const DocumentKey& key) {
   PumpEnqueuedLimboResolutions();
 }
 
+void SyncEngine::LoadBundle(std::shared_ptr<bundle::BundleReader> reader,
```

Comment:
Question: is this function supposed to run on our worker queue, or on the user executor?